book/opmanual_autolab/50_autolab_operation_manual/51_autolab_HW_checks_script.md

```diff
@@ -0,0 +1,117 @@
+# Autobot hardware compliance checks {#autolab-autobot-hw-checks status=ready}
```

Comment:
Shouldn't this part be in the minimal requirements after the fleet roster? I fell like putting it here is "too late" in the reader process, they have already read localization at this point in time...

Reply:
i put it there bc that way we have one "scripts" part. 
you can move it back if you want

Reply:
could you do it directly before we merge?